src/Microsoft.OData.Edm/Csdl/Semantics/CsdlSemanticsEntityTypeDefinition.cs

```diff
@@ -139,5 +161,30 @@ private IEnumerable<IEdmStructuralProperty> ComputeDeclaredKey()
 
             return null;
         }
+
+        private IEnumerable<KeyValuePair<IEdmStructuralProperty, string>> ComputeDeclaredKeyAlias()

```

Comment:
What does this method do?

Reply:
This method reads DeclaredKeyAliases from the <code>CsdlPropertyReference <code> object.